### PR TITLE
Feature/outbound links

### PR DESCRIPTION
**Pull request by starsinmypockets**
Related to https://app.zenhub.com/workspaces/datagov-multi-tenant-ckan-5cf12b601437ff1e943f0aa2/issues/gsa/datagov-ckan-multi/465

### CODE REVIEW

**Review by kmanaseryan — CHANGES_REQUESTED** [review]
Please check my comments above. Also I've run `yarn test`, it didn't pass. I think you should update the test snapshots and then see if the tests pass. Just for our note for the future, we should include yarn tests in the CI, so the tests won't pass if we have such case.

image

File: metadata-app/src/components/AdditionalMetadata/index.js [hunk 1]
```diff
@@ -32,9 +44,14 @@ const helpTexts = {
...
       about Exhibit 53 documents at{' '}
-      <a href="https://www.whitehouse.gov/wp-content/uploads/2018/06/fy-2020-it-budget-guidance.pdf">
+      <a
+        target="_blank"
+        rel="noopener noreferrer"
+        href="https://www.whitehouse.gov/wp-content/uploads/2018/06/fy-2020-it-budget-guidance.pdf"
+      >
         this pdf
       </a>
+      <OutboundLink height="15px" />
```
[the remark this answers is not in the document]
Comment (osahon-okungbowa) [reply metadata-app/src/components/AdditionalMetadata/index.js:54]: @kmanaseryan and @starsinmypockets i agree it's best if the icon is part of the link. However, the issue linked to this PR contains a link regarding the Accessibility standard. Pls Paul visit the link to confirm if including the icon as part of the link is acceptable.
Paul this is top priority pls. Update for merge **before**client meeting

File: metadata-app/src/components/RequiredMetadata/index.js [hunk 2]
```diff
@@ -5,9 +5,21 @@ import WrappedField from '../WrappedField';
 import api from '../../api';
 import Autocomplete from '../Autocomplete';
 import { ReactComponent as Info } from '../../img/info.svg';
+import Outbound from '../../img/outbound.svg';
 import HelpText from '../HelpText';
 import Radio from '../Radio';
 
+const OutboundLink = (props) => (
```
[the remark this answers is not in the document]
Comment (starsinmypockets) [reply metadata-app/src/components/RequiredMetadata/index.js]: Resolved, see `/src/components/LinkIcon/`

File: metadata-app/src/components/AdditionalMetadata/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (starsinmypockets) [reply metadata-app/src/components/AdditionalMetadata/index.js]: Resolved

**Review by kmanaseryan — APPROVED** [review]
LGreatTM 👍

**Pull request merged**